snippets/language-svg.cson

```diff
@@ -1,101 +1,352 @@
+    'descriptionMoreURL': 'https://developer.mozilla.org/de/docs/Web/SVG/Element/use'
 '.text.xml.svg .meta.tag.xml':
+  'clip-path':
+    'prefix': 'clip-path'
+    'body': 'clip-path="url(#$1)"'
+    'description': 'The clip-path attribute bind the element is applied to with a given <clippath> element'
+    'descriptionMoreURL': 'https://developer.mozilla.org/en-US/docs/Web/SVG/Attribute/clip-path'
+  'color-interpolation-filters':
+    'prefix': 'color-interpolation-filters'
+    'body': 'color-interpolation-filters="${1:sRGB}"'
+    'description': 'The color-interpolation-filters attribute specifies the color space for imaging operations performed via filter effects. Note that color-interpolation-filters has a different initial value than color-interpolation. color-interpolation-filters has an initial value of linearRGB, whereas color-interpolation has an initial value of sRGB. Thus, in the default case, filter effects operations occur in the linearRGB color space, whereas all other color interpolations occur by default in the sRGB color space.'
+    'descriptionMoreURL': 'https://developer.mozilla.org/en-US/docs/Web/SVG/Attribute/color-interpolation-filters'
+  'mask':
+    'prefix': 'mask'
+    'body': 'mask="url(#$1)"'
+    'description': 'The mask attribute bind the element is applied to with a given <mask> element'
+    'descriptionMoreURL': 'https://developer.mozilla.org/en-US/docs/Web/SVG/Attribute/mask'
+  'matrix':
+    'prefix': 'ma'
+    'body': 'transform="matrix($1, $2, $3, $4, $5, $6)"'
+    'description': 'This transform definition specifies a transformation in the form of a transformation matrix of six values.'
+    'descriptionMoreURL': 'https://developer.mozilla.org/de/docs/Web/SVG/Attribute/transform'
+  'height':
+    'prefix': 'h'
+    'body': 'height="$1"'
+    'description': 'This attribute indicates a vertical length in the user coordinate system. The exact effect of this coordinate depends on each element. Most of the time, it represents the height of the rectangular region of the reference element (see each individual element\'s documentation for exceptions).'
+    'descriptionMoreURL': 'https://developer.mozilla.org/de/docs/Web/SVG/Attribute/height'
+  'image-rendering':
+    'prefix': 'image-rendering'
+    'body': 'image-rendering="${1:auto}"'
```

Comment:
default it to 'pixelated' instead, that seems to be what people want to use when using image-rendering. 'auto' is the default anyway, so it'd be the same as not having specified anything.